frontend/pkg/frontend/otel_sdk.go

```diff
@@ -0,0 +1,88 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// TODO: can we move this into ocm-sdk-go/tracing?
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"go.opentelemetry.io/contrib/exporters/autoexport"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/trace"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/sdk/resource"
+)
+
+func InstallOpenTelemetryTracer(ctx context.Context, logger *slog.Logger, resourceAttrs ...attribute.KeyValue) (
+	func(context.Context) error,
+	error,
+) {
+	logger.InfoContext(ctx, "initialising OpenTelemetry tracer")
```

Comment:
(suggestion) would it be possible to log if the span exporter is a no-op exporter? E.g. either it's a noopSpanExporter instance or autoexport.IsNoneSpanExporter(...) returns true.